packages/reporter-vue/src/commands/CommandMessage.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <span v-if="message" class="message" :class="{ scaled }"><p>{{ message }}</p></span>
+  <Markdown v-else-if="markdown" :source="markdown" :class="{ scaled }" class="message text-warm-gray-400 italic markdown" />
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType, computed } from 'vue'
+import Markdown from 'vue3-markdown-it'
+
+export default defineComponent({
+  props: ['message', 'markdown', 'event'],
+  components: { Markdown },
+  setup(props) {
+    return {
+      scaled: computed(() => (props.message || props.markdown).length > 100)
```

Comment:
will `message` or `markdown` *always* exists? Can they be undefined?

Reply:
message/markdown are the text on a command... So... perhaps if someone doesn't normalize it correctly. Wouldn't hurt to throw an `|| ''` in there